Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardItemRow.swift

```diff
@@ -71,15 +78,27 @@ struct CreditCardItemRow: View {
             .padding(.trailing, 16)
             .padding(.top, 11)
             .padding(.bottom, 11)
-
+            .background(isTapped ? backgroundHoverColor : backgroundColor)
+            .gesture(
+                TapGesture()
+                    .onEnded { _ in
+                        didSelectAction?()
+                    }
+                    .simultaneously(with: DragGesture(minimumDistance: 0))
+                    .updating($isTapped) { (_, isTapped, _) in
+                        isTapped = true
+                    }
+            )
```

Comment:
Q. Why multi-gesture?
A. SwiftUI doesn't work properly to register taps and isn't fast to update isTapped variable based on simple tap gestures. We are left with a case where a user has tapped but colour doesn't change and this is where drag with distance 0 immediately updates the background color for hover case.